boost-1.48.0-add-bjam-man-page.patch

```diff
@@ -0,0 +1,150 @@
+diff --git a/tools/build/v2/doc/bjam.1 b/tools/build/v2/doc/bjam.1
```

Comment:
Again, what do you need this manpage for?

Reply:
> Again, what do you need this manpage for?

I don't see any reason to reject patches that are already available and that don't spoil anything.
What's more, the fact that now flatpak doesn't export man pages doesn't mean that it won't happen in the future.